src/main/resources/public/peer.js

```diff
@@ -17,7 +19,8 @@ const elements = [
   "otherVideo",
   "errorOverlay",
   "errorMessage"
-].map(e => document.getElementById(e))
+]
+elementIds.forEach(e => (elements[e] = document.getElementById(e)))
```

Comment:
`map` doesn't work here?

Reply:
That's the reason it didn't work in the first place (completely empty page). I'll check if we have another solution